bukkit/src/main/resources/plugin.yml

```diff
@@ -9,28 +9,122 @@ api-version: '1.13'
 prefix: AuraSkills
 
 permissions:
-    auraskills.command.skills:
+    auraskills.*:
+        description: Permission to use all commands from AuraSkills
+        children:
+            auraskills.command.user: true
+            auraskills.skill.*: true
+            auraskills.command.admin: true
+        default: false
+
+    auraskills.command.user:
+        description: Permissions users have by default
+        children:
+            auraskills.command.skills: true
+            auraskills.command.stats: true
+            auraskills.command.sources: true
+            auraskills.command.help: true
+            auraskills.command.mana: true
+            auraskills.command.top: true
+            auraskills.command.rank: true
+            auraskills.command.lang: true
+            auraskills.command.abtoggle: true
+            auraskills.command.claimitems: true
         default: true
-    auraskills.command.stats:
+
+    auraskills.skill.*:
+        description: Permission to use all skills
+        children:
+            auraskills.skill.farming: true
+            auraskills.skill.foraging: true
+            auraskills.skill.mining: true
+            auraskills.skill.fishing: true
+            auraskills.skill.excavation: true
+            auraskills.skill.archery: true
+            auraskills.skill.defense: true
+            auraskills.skill.fighting: true
+            auraskills.skill.endurance: true
+            auraskills.skill.agility: true
+            auraskills.skill.alchemy: true
+            auraskills.skill.enchanting: true
+            auraskills.skill.sorcery: true
+            auraskills.skill.healing: true
+            auraskills.skill.forging: true
         default: true
+
+    auraskills.command.itemmaker:
```

Comment:
Rename this to auraskills.command.item.*

Reply:
then i am putting these

```yml
auraskills.command.armor.modifier: true
auraskills.command.armor.requirement: true
auraskills.command.armor.multiplier: true
```

into their own `auraskills.command.armor.*`  for clarity